programs/survival/horse_stats.sc

```diff
@@ -0,0 +1,24 @@
+// Print the stats of the horse the player is looking at
+
+__config() -> ({ 'stay_loaded' -> true });
```

Comment:
suggest removing the config - it anyone wants that to load automatically they can add that.

Reply:
done